bundles/org.openhab.binding.hdpowerview/src/test/java/org/openhab/binding/hdpowerview/HDPowerViewJUnitTests.java

```diff
@@ -100,9 +99,8 @@ public void testOnlineCommunication() {
 
         if (VALID_IP_V4_ADDRESS.matcher(hubIPAddress).matches()) {
             // initialize stuff
-            Client client = ClientBuilder.newClient();
+            HttpClient client = new HttpClient();
             assertNotNull(client);
-            // client.register(new Logger());
```

Comment:
Did you try `client.property("org.apache.cxf.logging.FaultListener", <customFaultListener>);`? I don't know for sure if would work though. But if it did I think it would be a simpler way to solve this particular issue.

Reply:
To be honest, I played around with Apache CXF for a whole day (it is not well documented), and I couldn't get it to work. Which is why I decided to go for the Jetty client solution: it required more lines of coding, but it required less time.

Reply:
Understood. I don't know why the cxf folks implemented a standard api but refused to support the configuration standard of that api.